go/appencryption/pkg/kms/aws.go

```diff
@@ -281,11 +281,13 @@ func (m *AWSKMS) DecryptKey(ctx context.Context, keyBytes []byte) ([]byte, error
 			decryptKeyTimer.UpdateSince(start)
 
 			if err != nil {
+				log.Printf("error kms decrypt: %s", err)
```

Comment:
Need to use [`Debugf`](https://github.com/godaddy/asherah/blob/a6c6eacc25a5b6a020b9f8e6242c090ced4ff7a6/go/appencryption/pkg/log/log.go#L18) from the internal `log` package to ensure these are only logged when debugging is enabled.